tests/gtest/aviftest_helpers.cc

```diff
@@ -37,16 +67,16 @@ void fillImagePlain(avifImage * image, const uint32_t yuva[4])
     avifPixelFormatInfo info;
     avifGetPixelFormatInfo(image->yuvFormat, &info);
 
-    for (int c : AVIF_CHANS) {
-        uint8_t * row = (c == AVIF_CHAN_A) ? image->alphaPlane : image->yuvPlanes[c];
+    for (int c : AVIF_YUV_CHANS) {
```

Comment:
Nit/optional: We can write this as a traditional for loop:

```
for (int c = 0; c < 4; c++) {
```

Since `c` is used to index the `image->yuvPlanes` and `image->yuvRowBytes` arrays, we must assume `c` is 0, 1, or 2 (when `c` is not the alpha channel).